package.json

```diff
@@ -6,18 +6,12 @@
   "license": "MIT",
   "scripts": {
     "prepare": "husky",
-    "copy-views": "cp -R server/views dist/server/",
-    "compile-sass": "sass --quiet-deps --no-source-map --load-path=node_modules/govuk-frontend/dist --load-path=node_modules/@ministryofjustice/frontend --load-path=. server/assets/scss/application.scss:./dist/server/assets/stylesheets/application.css --style compressed",
     "watch-ts": "tsc -w",
-    "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
-    "watch-node": "DEBUG=gov-starter-server* nodemon -r dotenv/config --watch dist/ dist/server.js | bunyan -o short",
-    "watch-sass": "npm run compile-sass -- --watch",
-    "build": "npm run compile-sass && tsc && npm run copy-views",
+    "build": "NODE_ENV=production && node esbuild/esbuild.config.js --build",
```

Comment:
why force the environment to always be production? the Dockerfile sets `NODE_ENV` so the deployed version would already end up compiling in production mode